pubsub/pubsub_goredis.go

```diff
@@ -25,6 +28,7 @@ type GoRedisPubSub struct {
 	Config  config.Config   `inject:""`
 	Logger  logger.Logger   `inject:""`
 	Metrics metrics.Metrics `inject:"metrics"`
+	Tracer  trace.Tracer    `inject:"tracer"`
```

Comment:
If we don't expect to inject multiple different tracers, we don't need to name them. You could remove the name here, and in main where it's created.